source/administration/command-line-tools.rst

```diff
@@ -524,6 +560,31 @@ mattermost config get
     .. code-block:: none
 
           --path string  Optional subpath; defaults to value in Site URL.
+	  
+mattermost config migrate
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Migrate a file-based configuration to (or from) a database-based configuration. Point the Mattermost server at the target configuration to start using it. If using SAML, ensure the SAML certificates and keys are accessible to also migrate into the database.
+    
+    .. note::    
+    If a ``from`` parameter is not specified, the command will fall back to what is specified in --config.
+
+  Format
+    .. code-block:: none
+
+      mattermost config migrate {config to read} {config to write}
+
+  Examples
+    .. code-block:: none
+
+       ./mattermost config migrate  path/to/config.json "postgres://mmuser:mostest@dockerhost:5432/mattermost_test?sslmode=disable&connect_timeout=10"
+
+ Options
```

Comment:
The options section may need updating as well

Reply:
Good catch -thanks.  I've updated